plugins/reporters/asciidoc/src/funTest/kotlin/ManPageTemplateReporterFunTest.kt

```diff
@@ -39,7 +41,10 @@ class ManPageTemplateReporterFunTest : StringSpec({
 
         reportContent should matchExpectedResult(
             expectedResultFile,
-            custom = mapOf("<REPLACE_DATE>" to "${LocalDate.now()}")
+            custom = mapOf(
+                "<REPLACE_DATE>" to "${LocalDate.now()}",
+                "<REPLACE_ASCII_DOCTOR_VERSION>" to Asciidoctor.Factory.create().asciidoctorVersion()
```

Comment:
How about instead making `AsciiDocTemplateReporter.asciidoctor` `internal` and using it here? In the same run `ManPageTemplateReporter()` would probably be assigned to a variable for reuse here.